cc/toolchains/args/generate_linkmap/BUILD

```diff
@@ -0,0 +1,22 @@
+load("//cc/toolchains:args.bzl", "cc_args")
+load("//cc/toolchains:feature.bzl", "cc_feature")
+
+package(default_visibility = ["//visibility:public"])
+
+cc_feature(
+    name = "feature",
+    args = [":generate_linkmap"],
+    feature_name = "generate_linkmap",
+)
+
+cc_args(
+    name = "generate_linkmap",
+    actions = ["//cc/toolchains/actions:link_actions"],
```

Comment:
this doesn't match today's feature, but is an attempt to future proof it since ideally this would work with all link actions, and today bazel gates it so it only applies to binaries